lib/process-services-cloud/src/lib/process/process-list/services/process-list-cloud.service.ts

```diff
@@ -63,6 +69,32 @@ export class ProcessListCloudService extends BaseCloudService {
         }
     }
 
+    private getAdminProcessByRequest(queryUrl: string, queryParams: any): Observable<any> {
+        const postBody = {
+            variableKeys: this.getVariableKeysFromQueryParams(queryParams)
+        };
+
+        delete queryParams['variableKeys'];
+
+        return this.post(queryUrl, postBody, queryParams).pipe(
```

Comment:
a test to check that the `post` method is called with the right params is missing